backend/serverless.yml

```diff
@@ -70,19 +96,11 @@ functions:
           route: $disconnect
   read:
     handler: functions/read.main
-    iamRoleStatementsInherit: true
-    iamRoleStatements:
-      - Effect: Allow
-        Action:
-          - dynamodb:Query
-        Resource:
-          - 'Fn::GetAtt': [RoomTable, Arn]
```

Comment:
why remove this ? If someone disconnects, then we need to clean his `watcher` key/value in `room.watchers` no ?

Reply:
Yes we'll need the dynamodb:PutItm Action. too. For now it's not useful.